Add descriptions to MDX frontmatter (#614)
vercel/turborepo: docs/pages/docs/getting-started.mdx

---
title: Getting Started

> description: Learn how to get started with Turborepo. Create a new starter monorepo or add Turborepo to an existing project.

Reviewer: ```suggestion
description: Create your first monorepo or add Turborepo to an existing project.
```
